packages/appstore/src/components/modals/real-wallets-upgrade/components/modal-elements/desktop-real-wallets-upgrade.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { Modal } from '@deriv/components';
+import { useStore, observer } from '@deriv/stores';
+import WalletsUpgradeFooter from './wallets-upgrade-footer';
+import WalletsUpgradeContent from './wallets-upgrade-content';
+import { TRealWalletsUpgrade } from 'Types';
+
+const DesktopRealWalletsUpgrade = observer(
+    ({
+        current_step,
+        handleBack,
+        handleClose,
+        handleNext,
+        is_disabled,
+        toggleCheckbox,
+        upgradeToWallets,
+    }: TRealWalletsUpgrade) => {
+        const { traders_hub: is_real_wallets_upgrade_on } = useStore();
+        return (
+            <Modal
+                is_open={is_real_wallets_upgrade_on}
+                toggleModal={handleClose}
+                height='734px'
+                width='1200px'
+                should_header_stick_body={false}
+                has_close_icon
+                title=' '
```

Comment:
hey @amina-deriv 
believe that it was done because we need to show `close cross button`. our current logic of `Modal` component not allow us to do this without sending any title/header/renderTitle prop. I did small refactoring/improvement of `Modal` component. Now if we want to render just `close cross button` we can only pass the `has_close_icon` prop and it will render header with `close cross button`
cc: @aizad-deriv